packages/core/src/Stores/notification-store.js

```diff
@@ -1078,6 +1082,16 @@ export default class NotificationStore extends BaseStore {
                 message: <Localize i18n_default_text='Please log in with your updated password.' />,
                 type: 'info',
             },
+            phone_number_verification: {
+                key: 'phone_number_verification',
+                header: localize('Verify your phone number'),
+                message: <Localize i18n_default_text='Keep your account safe. Verify your phone number now.' />,
+                type: 'warning',
+                action: {
+                    route: routes.personal_details,
```

Comment:
shouldnt we route to routes.phone_verification

Reply:
Yeaa will change route to phone verification